src/components/overlays/WindowManager.vue

```diff
@@ -0,0 +1,57 @@
+<!--
+//  WindowManager.vue
+//
+//  Created by Kalila L. & Heather Anderson on May 14th, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+-->
+
+<template>
+    <div v-for="overlay in overlays" :key="overlay">
+        <component :is="overlay" @overlay-action="onAction(overlay, $event)" />
+    </div>
+</template>
+
+<script>
+export default {
```

Comment:
Add the export name, to make it more obvious what's being exported?

Can the export statement be down the bottom, somehow? (Similarly, elsewhere.)

Reply:
... Seems like this is a Vue default template thing.

Reply:
Yah, the only component of the <script> here is a single export of "default" being an object definition describing the component.